README.md

```diff
@@ -1,2 +1,29 @@
-# Portfolio-setup-and-mobile-version-skeleton
-My Protfolio
+# Project Name
+
+Portfolio-setup-and-mobile-version-skeleton
+
+# PR
+
+basic design of first section in my portfolio
+
+![screenshot](./Capture.png)
+
+The very first siction of the mobile verstion of my portfolio
+
+## Built With
+
+- Html
+- css
+- Litner
+
+### Prerequisites
+
+Chrome or any explorer
+
+## Authors
+
+👤 **Meqdam Al-qudah**
+
+- GitHub: https://github.com/MeqdamAlqudah
+- Twitter:https://twitter.com/MeqdamQudah
+- LinkedIn:www.linkedin.com/in/meqdam-al-qudah-7514a21b5
```

Comment:
- [ ] **[OPTIONAL]** Perhaps you can format the links here to be neater. You can do the following:
```
- [GitHub](https://github.com/MeqdamAlqudah)
- [Twitter](https://twitter.com/MeqdamQudah)
- [LinkedIn](www.linkedin.com/in/meqdam-al-qudah-7514a21b5)
```